adserver/management/commands/archive_offers.py

```diff
@@ -177,30 +172,26 @@ def delete_offers(self, start_day, end_day):
             )
             return
 
-        self.stdout.write(_("Deleting archived offers..."))
+        self.stdout.write(_("Deleting archived offers for %s...") % day)
```

Comment:
This makes me less worried, since it's more bounded 👍

Reply:
I probably still won't run it with that option until this whole process is very smooth.